lib/local_contact_importer.rb

```diff
@@ -17,12 +17,9 @@ def process_row(row)
     authority.name = decode_broken_entities( row['Name'] )
     authority.contact_address = parse_address(row)
     authority.contact_phone = decode_broken_entities( row['Telephone Number 1'] )
-    url = row['Contact page URL']
-    unless url.blank? || url.start_with?("http://") || url.start_with?("https://")
-      url = "http://" + url
-    end
-    authority.contact_url = url
+    authority.contact_url = parse_url( row['Contact page URL'] )
     authority.contact_email = row['Main Contact Email']
+    authority.homepage_url = parse_url( row['Home page URL'] )
```

Comment:
Human linter: could you remove the spaces between the brackets as per the style guide?